app/javascript/Components/assignment_chart.jsx

```diff
@@ -187,6 +188,28 @@ export class AssignmentChart extends React.Component {
         );
       }
 
+      let annotation_summary = (
+        <div className="distribution-graph">
+          <h3>{I18n.t("assignments.annotation_summary")}</h3>
+          <p>
+            {I18n.t("assignments.average_annotations", {
+              average_annotations: this.state.summary.average_annotations,
```

Comment:
This isn't quite going to work if the state is `null`; you can use `||` to set a default value of 0